src/app/views/infoblock/infoblock.js

```diff
@@ -0,0 +1,48 @@
+class InfoblockController {
+
+  /** @ngInject */
+  constructor($log) {
+    this.$log = $log;
+  }
+
+  $onChanges() {
+    if (!this.variable) {
+      return;
+    }
+
+    this.value = this.variable.value;
+    this.title = this.variable.title;
+
+    this.colorClass = '';
+    if (this.colorOverride) {
+      this.colorClass = `color-${this.colorOverride}`;
+    } else if (this.isChange && this.value !== 0) {
+      let ispositive = (this.value > 0);
+      if (this.reverseColor) {
+        ispositive = !ispositive;
+      }
+      this.colorClass = ispositive ? 'color-positive' : 'color-negative';
+    }
+
+    this.value = Math.round(this.value * 100) / 100;
```

Comment:
Doing these computations from here on in this method is fine, but it may make sense to use a filter instead (or write your own for the custom logic).

Either way, the values that are not percents and not changes should be piped through the number filter so that they're automatically formatted for the browser locale (adds things like commas for larger numbers, etc).

https://docs.angularjs.org/api/ng/filter/number